app/detail.tsx

```diff
@@ -124,13 +120,29 @@ export function AlphabetCardDetailScreen({
         );
     };
 
-    const letterAudioSource = `${BASE_API_URL}/resources/mediaitems/download?name=${letter_audio}`;
-
-    const wordAudioSource = `${BASE_API_URL}/resources/mediaitems/download?name=${word_audio}`;
-
-    const player = useAudioPlayer(letterAudioSource);
+    const animatedValue = useRef(new Animated.Value(0)).current;
 
-    const player2 = useAudioPlayer(wordAudioSource);
+    useEffect(() => {
+        Animated.sequence([
+            Animated.timing(animatedValue, {
+                toValue: 5,
+                duration: 600,
+                easing: Easing.inOut(Easing.ease),
+                useNativeDriver: true,
+            }),
+            Animated.timing(animatedValue, {
+                toValue: 0,
+                duration: 500,
+                easing: Easing.inOut(Easing.ease),
+                useNativeDriver: true,
+            }),
+        ]).start();
+    }, [animatedValue]);
+
+    const translateX = animatedValue.interpolate({
```

Comment:
I wonder if you should find a wrapper pattern where you isolate this animation and also make it reusable. It doesn't matter for a small application, but if you are porting all of the COSCRAD client across, you'd want to be sure that all the animations are the same.

Reply:
I created a component for this animation.